src/main/java/se/sundsvall/chat/api/model/Conversation.java

```diff
@@ -0,0 +1,171 @@
+package se.sundsvall.chat.api.model;
+
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.List;
+import java.util.Objects;
+
+@Schema(description = "Represents a conversation containing participants, messages, and metadata.")
+public class Conversation {
+
+	@Schema(description = "The unique identifier of the conversation.", example = "1234567890")
+	private String id;
+
+	@ArraySchema(schema = @Schema(implementation = Participant.class, description = "The list of participants in the conversation."))
+	private List<Participant> participants;
+
+	@Schema(description = "The identifier of the municipality associated with the conversation.", example = "municipality-123")
+	private String municipalityId;
+
+	@Schema(description = "The namespace of the conversation.", example = "namespace-abc")
+	private String namespace;
+
+	@Schema(description = "A list of channel, reference, and context identifiers associated with the conversation.",
+		example = "[\"channel1-ref1-context1\", \"channel2-ref2-context2\"]")
+	private String channelId;
+
+	@ArraySchema(schema = @Schema(implementation = MetaData.class, description = "A list of metadata associated with the conversation."))
+	private List<MetaData> metaData;
+
+	@Schema(description = "The topic of the conversation.", example = "Customer Support")
+	private String topic;
+
+	@Schema(description = "The list of messages in the conversation.")
+	private List<Message> messages;
```

Comment:
Should we have messages here (in the API-model)? 
We have a dedicated resource for messages.

Reply:
No.